service/storage_proxy.cc

```diff
@@ -433,51 +535,72 @@ class storage_proxy::remote {
         errors_info errors;
         ++p->get_stats().received_mutations;
         p->get_stats().forwarded_mutations += forward.size();
-        co_await coroutine::all(
-            [&] () -> future<> {
-                try {
-                    // FIXME: get_schema_for_write() doesn't timeout
-                    schema_ptr s = co_await get_schema_for_write(schema_version, netw::messaging_service::msg_addr{reply_to, shard});
-                    // Note: blocks due to execution_stage in replica::database::apply()
-                    co_await apply_fn(p, trace_state_ptr, std::move(s), m, timeout);
-                    // We wait for send_mutation_done to complete, otherwise, if reply_to is busy, we will accumulate
-                    // lots of unsent responses, which can OOM our shard.
-                    //
-                    // Usually we will return immediately, since this work only involves appending data to the connection
-                    // send buffer.
-                    auto f = co_await coroutine::as_future(send_mutation_done(netw::messaging_service::msg_addr{reply_to, shard}, trace_state_ptr,
+
+        const auto scr_addr = netw::messaging_service::msg_addr{reply_to, shard};
```

Comment:
Why move it here?

Reply:
it's used in two places now, in the old one, and in `write_request_info`

Reply:
My Firefox search does not find the second use, though I do see it now.

Reply:
Ah, this is because there is a typo here! scr_addr vs src_addr. Does it compile?